content/v2.0/influxdb-templates/stacks/update.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Update an InfluxDB stack
+list_title: Update a stack
+description: >
+  Use the [`influx apply` command](/v2.0/reference/cli/influx/apply/)
+  to update a stack with a modified template.
```

Comment:
to apply a modified template to the stack?

Reply:
The description is what shows up in google search results. I'm trying to keep the solution at the beginning of the sentence so it's not lost when google truncates the summary.